src/app/core/sidebar/sidebar.component.ts

```diff
@@ -17,20 +17,24 @@ import { SideContentComponent } from '../side-content/side-content.component';
     MatSidenavModule,
     MatListModule,
     SideContentComponent,
-    CommonModule
+    CommonModule,
+    RouterOutlet
   ],
   templateUrl: './sidebar.component.html',
   styleUrl: './sidebar.component.scss'
 })
 export class SidebarComponent implements OnInit { 
   @Input() changing!: Subject<boolean>;
   @Input() isMobile = false;
-  @ViewChild(MatSidenav)
-  sidenav!: MatSidenav;
+  @ViewChild(MatSidenav) sidenav!: MatSidenav;
   isCollapsed = true;
 
+  constructor(private router: Router) {
+
+  }
+
   ngOnInit(){
-    this.changing.subscribe(v => { 
+    this.changing.subscribe(() => { 
```

Comment:
@SelmaAhmetovic , Da li je uradjen unsubscribe kako bi se sprijecilo curenje memorije?

Reply:
will be done